src/modules/Remote.rb

```diff
@@ -135,18 +149,25 @@ def Read
         from: "any",
         to:   "list <map>"
       )
-      vnc_conf = Builtins.filter(xinetd_conf) do |m|
-        s = Ops.get_string(m, "service", "")
-        s == "vnc1" || s == "vnchttpd1"
+
+      xinetd_vnc1_enabled = false
+      xinetd_vnchttp1_enabled = false
```

Comment:
These assignments seems to be useless. Default value for uninitialized variables is ```nil``` and it is treated as ```false``` in ruby.